examples/0_basic.jl

```diff
@@ -86,38 +85,58 @@ We represent it using a type called `ImplicitFunction`, which you will see in ac
 =#
 
 #=
-First we define a `forward` pass correponding to the function we consider.
-It returns the actual output $y(x)$ of the function, as well as the optional byproduct $z(x)$.
-Here we don't need any additional information, so we set $z(x)$ to $0$.
-Importantly, this forward pass _doesn't need to be differentiable_.
+First we define a forward mapping correponding to the function we consider.
+It returns the actual output $y(x)$ of the function, and can be thought of as a black box solver.
+Importantly, this Julia callable _doesn't need to be differentiable by automatic differentiation packages but the underlying function still needs to be mathematically differentiable_.
```

Comment:
I stressed the difference between the Julia callable and the mathematical function